staubli_tx2_60_gazebo/urdf/tx2_60_macro.xacro

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+<xacro:include filename="$(find staubli_tx2_60_support)/urdf/tx2_60_macro.xacro"/>
+<xacro:macro name="staubli_tx2_60_robot" params="prefix">
+
+  <!-- get URDF description of Staubli TX60 model -->
+  <xacro:staubli_tx2_60 prefix="${prefix}" />
+
+  <!-- transmission list -->
+  <transmission name="${prefix}tran1">
+    <robotNamespace>/${prefix}</robotNamespace>
+    <type>transmission_interface/SimpleTransmission</type>
+    <joint name="${prefix}joint_1">
+      <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
```

Comment:
We may want to discuss whether using `EffortJointInterface` would be better.

It would mean we'd have to tune pids, but interaction with other Gazebo simulated entities would be more robust and of higher quality (position controlled robots in Gazebo are "infinitely stiff", leading to potentially exploding robots when they come into contact with objects).

Reply:
Sure, I just followed the existing method in other gazebo packages in this repository. So i did not bother to change that. I will try out  `EffortJointInterface` instead of `PositionJointInterface`

Reply:
@gavanderhoorn  are you aware of ros-industrial repos that makes use of  `EffortJointInterface`

Reply:
Yes, `abb_irb1200_gazebo` (see [here](https://github.com/ros-industrial/abb_experimental/blob/04aa3e515c5672cc216180901c59b39dbde99258/abb_irb1200_gazebo/urdf/irb1200_5_90_macro.xacro)).

There's nothing really to it btw, just replace `PositionJointInterface` with `EffortInterface` and start a different controller.

Reply:
Yes, `abb_irb1200_gazebo` (see [here](https://github.com/ros-industrial/abb_experimental/blob/04aa3e515c5672cc216180901c59b39dbde99258/abb_irb1200_gazebo/urdf/irb1200_5_90_macro.xacro)).

There's nothing really to it btw, just replace `PositionJointInterface` with `EffortInterface` and start a different controller.